autoload/grepper.vim

```diff
@@ -455,6 +455,7 @@ function! s:finish_up(flags) abort
     nnoremap <silent><buffer> <cr> <cr>zv
     nnoremap <silent><buffer> o    <cr>zv
     nnoremap <silent><buffer> O    <cr>zv<c-w>p
+    nnoremap <silent><buffer> O    :let w=winnr()<cr><cr>zv:exec w 'wincmd p'<cr>
```

Comment:
Sorry, forgot to remove the original mapping: https://github.com/mhinz/vim-grepper/pull/49.